vendor/bat-native-ledger/include/bat/ledger/ledger_client.h

```diff
@@ -222,6 +224,10 @@ class LEDGER_EXPORT LedgerClient {
 
   virtual void GetExcludedPublishersNumberDB(
       ledger::GetExcludedPublishersNumberDBCallback callback) = 0;
+
+  virtual void GetCountryCodes(
+      const std::vector<std::string>& limited_countries,
```

Comment:
Maybe more general param? Like `countries`, as this function could be used by other functions as well, where we wouldn't passing limited countries in

Reply:
Done.